bin/ConcordanceCheck.sh

```diff
@@ -0,0 +1,223 @@
+
+======================================================================================================================
+
+EOH
+        trap - EXIT
+        exit 0
+}
+
+
+#
+##
+### Main.
+##
+#
+
+#
+# Get commandline arguments.
+#
+log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "Parsing commandline arguments..."
+declare group=''
+while getopts "g:l:h" opt
+do
+        case $opt in
+                h)
+                        showHelp
+                        ;;
+                g)
+                        GROUP="${OPTARG}"
+                        ;;
+                l)
+                        l4b_log_level=${OPTARG^^}
+                        l4b_log_level_prio=${l4b_log_levels[${l4b_log_level}]}
+                        ;;
+                \?)
+                        log4Bash "${LINENO}" "${FUNCNAME:-main}" '1' "Invalid option -${OPTARG}. Try $(basename $0) -h for help."
+                        ;;
+                :)
+                        log4Bash "${LINENO}" "${FUNCNAME:-main}" '1' "Option -${OPTARG} requires an argument. Try $(basename $0) -h for help."
+                        ;;
+        esac
+done
+
+#
+# Check commandline options.
+#
+if [[ -z "${GROUP:-}" ]]
+then
+        log4Bash 'FATAL' "${LINENO}" "${FUNCNAME:-main}" '1' 'Must specify a group with -g'
+fi
+
+#
+# Source config files.
+#
+log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "Sourcing config files..."
+declare -a configFiles=(
+        "${CFG_DIR}/${GROUP}.cfg"
+        "${CFG_DIR}/${HOSTNAME_SHORT}.cfg"
+        "${CFG_DIR}/sharedConfig.cfg"
+        "${HOME}/molgenis.cfg"
+)
+
+for configFile in "${configFiles[@]}"; do 
+        if [[ -f "${configFile}" && -r "${configFile}" ]]
+        then
+                log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "Sourcing config file ${configFile}..."
+                #
+                # In some Bash versions the source command does not work properly with process substitution.
+                # Therefore we source a first time with process substitution for proper error handling
+                # and a second time without just to make sure we can use the content from the sourced files.
+                #
+                mixed_stdouterr=$(source ${configFile} 2>&1) || log4Bash 'FATAL' ${LINENO} "${FUNCNAME:-main}" ${?} "Cannot source ${configFile}."
+                source ${configFile}  # May seem redundant, but is a mandatory workaround for some Bash versions.
+        else
+                log4Bash 'FATAL' "${LINENO}" "${FUNCNAME:-main}" '1' "Config file ${configFile} missing or not accessible."
+        fi
+done
+
+#
+# Make sure to use an account for cron jobs and *without* write access to prm storage.
+#
+
+if [[ "${ROLE_USER}" != "${ATEAMBOTUSER}" ]]
+then
+        log4Bash 'FATAL' "${LINENO}" "${FUNCNAME:-main}" '1' "This script must be executed by user ${ATEAMBOTUSER}, but you are ${ROLE_USER} (${REAL_USER})."
+fi
+
+
+module load HTSlib/1.3.2-foss-2015b
```

Comment:
Can we create a parameters/ config file for the scripts not generated by Compute? In that way we do not have hardcoded versions in our scripts.